src/test/java/edu/isi/oba/ObaUtilsTest.java

```diff
@@ -1,5 +1,7 @@
 package edu.isi.oba;
 
+import com.github.jknack.handlebars.internal.Files;
```

Comment:
Do we need this?

Reply:
no, will remove

Reply:
Fixed in https://github.com/KnowledgeCaptureAndDiscovery/OBA/pull/96/commits/d694104a837a295404aae79fb36b6aee43cd0524